src/index.js

```diff
@@ -44,7 +46,12 @@ export function createSelectorCreator(memoize, ...memoizeOptions) {
     const memoizedResultFunc = memoize(
       (...args) => {
         recomputations++
-        return resultFunc(...args)
+        try {
+          return resultFunc(...args)
+        } catch (e) {
+          errorSubscriptions.emitError(e, resultFunc, args, dependencies)
```

Comment:
Would there be any way to attach the list of dependencies directly to the error object? It would be nice to get these details without having to subscribe externally.